app-emacs/flycheck/flycheck-32_p20210825.ebuild

```diff
@@ -0,0 +1,21 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit elisp
+
+DESCRIPTION="Modern on-the-fly syntax checking extension for GNU Emacs"
+HOMEPAGE="https://www.flycheck.org/"
+SRC_URI="https://github.com/flycheck/flycheck/archive/784f184cdd9f9cb4e3dbb997c09d93e954142842.tar.gz -> ${P}.tar.gz"
```

Comment:
Could you put the commit hash into a variable and reference it in SRC_URI and S?